colossalai/tensor/chunk.py

```diff
@@ -236,10 +235,9 @@ def access_chunk(self, tensor: torch.Tensor) -> None:
         self.accessed_chunks.add(chunk)
         self.total_mem[chunk.device_type] += chunk.mem
 
-    def release_chunk(self, tensor: torch.Tensor) -> None:
+    def release_chunk(self, chunk: Chunk) -> None:
```

Comment:
will the API changing affects our old code?

Reply:
Chunk manager's methods should not be called by users directly. I think user's code won't be influenced by this PR.